### PR TITLE
Add dropdown entry for new job composer

### LINKED ISSUE #2618: add projects to `jobs` dropdown menu

### PR DESCRIPTION
**Pull request by Oglopf**
Fixes #2618

### CODE REVIEW

**Comment by Oglopf** [comment]
Fixes #2618

**Comment by Oglopf** [comment]
I am not sure if I should be using `job-composer` or `projects` for this app. I'm also not sure if this was exactly what you were describing or not in the issue as how to do this.

I also wasn't sure how to test this.

**Comment by johrstrom** [comment]
> please update the .deb package as well. That's a shell script - https://github.com/OSC/ondemand/blob/c38b2f288b0f9578edcec564447320a1c93b3a50/packaging/deb/rules - that is a shell script so it could be easier

Maybe you'll just need to `chmod 700` - looks like we move all the subdirectories here already.

https://github.com/OSC/ondemand/blob/c38b2f288b0f9578edcec564447320a1c93b3a50/packaging/deb/rules#L45

**Comment by johrstrom** [comment]
This looks OK - I just want to wait until we have 3.1 repos setup and we start publishing 3.1 nightlies.

**Comment by johrstrom** [comment]
Errr actually - I think we need the chmod 700 in the RPM spec too.

File: packaging/rpm/ondemand.spec [hunk 1]
```diff
@@ -265,6 +266,8 @@ touch %{_localstatedir}/www/ood/apps/sys/myjobs/tmp/restart.txt
 %{_localstatedir}/www/ood/apps/sys/file-editor
 %{_localstatedir}/www/ood/apps/sys/activejobs
 %{_localstatedir}/www/ood/apps/sys/myjobs
+%{_localstatedir}/www/ood/apps/sys/projects
+%attr(700, root, root) %{_localstatedir}/www/ood/apps/sys/projects
```
[the remark this answers is not in the document]
Comment (treydock) [reply packaging/rpm/ondemand.spec:270]: Yep, though could be really specific and make it `0700` but either way works.

**Review by johrstrom — APPROVED** [review]

**Pull request merged**